### DISCUSSION TITLE
env file comments

### DISCUSSION THREAD

**Discussion by gitmikebrown** [laravel/framework#52053] [Ideas]
Is there a reason there's no comments in the env.example file?  Everything in Laravel is so well commented on with this file being the exception.  Can comments be added so that it helps developers know what options are possible? Or, a link provided on where to go to find out more info?  As an example, add a comment section above the DB vars that contains info on what options are available for DB_CONNECTION.  Add a comment section above the SESSION vars to help understand what options are available for those too. And so on.

**Comment by browner12** [comment]
the variables are mostly commented in the `/config` directory files. I would recommend looking there for explanations.

**Reply by gitmikebrown** [reply]
Thank you for the reply.  I saw that, but for people who are new, how would they know that?  It's not forthcoming.

**Reply by browner12** [reply]
"Configuration" is part of the "Getting Started" docs. If you're new, you should be reading these sections.

**Reply by gitmikebrown** [reply]
https://laravel.com/docs/11.x/configuration#environment-configuration
I didn't see anything about the various options/values that can be applied to the vars in the env file.  For ex, there isn't anything in the configurations link that says to type mariadb, mysql, etc., for the DB_CONNECTION, or for any other vars and their options.  I think it would be worth it to add this information somewhere.  Either within the included configuration link, or in the env file itself.

**Reply by browner12** [reply]
Correct. the docs are only telling you *where* to look, which is in the `/config` files.

**Reply by gitmikebrown** [reply]
I looked and couldn't find the avaliable values for  'APP_ENV' other than 'production', or what changing this does.  Same with some of the other vars and their values.
